api/src/pcapi/core/external/compliance.py

```diff
@@ -0,0 +1,130 @@
+import logging
+
+import sqlalchemy as sa
+
+from pcapi import settings
+from pcapi.core.auth import api as auth_api
+from pcapi.core.offers import models as offers_models
+from pcapi.domain import music_types
+from pcapi.domain import show_types
+from pcapi.models import db
+from pcapi.tasks import compliance_tasks
+from pcapi.tasks.serialization.compliance_tasks import GetComplianceScoreRequest
+from pcapi.utils import requests
+
+
+logger = logging.getLogger(__name__)
+
+
+def update_offer_compliance_score(offer: offers_models.Offer, is_primary: bool, asynchronous: bool = True) -> None:
+    payload = _get_payload_for_compliance_api(offer)
+    if is_primary:
+        if asynchronous:
+            compliance_tasks.update_offer_compliance_score_primary_task.delay(payload)
+        else:
+            compliance_tasks.update_offer_compliance_score_primary_task(payload)
+    else:
+        if asynchronous:
+            compliance_tasks.update_offer_compliance_score_secondary_task.delay(payload)
+        else:
+            compliance_tasks.update_offer_compliance_score_secondary_task(payload)
+
+
+def make_update_offer_compliance_score(payload: GetComplianceScoreRequest) -> None:
+    offer = offers_models.Offer.query.get(payload.offer_id)
```

Comment:
Si la file d'attente de la _cloud task_ prend du retard et que l'offre est supprimée entre temps, alors on risque d'avoir un crash plus loin avec `offer = None`. Suggestion, ignorer ce cas : 
```suggestion
    offer = offers_models.Offer.query.get(payload.offer_id)
    if not offer:
        return
```